### DISCUSSION TITLE
Better Product Features

### DISCUSSION THREAD

**Discussion by harm-smits** [PrestaShop/PrestaShop#40195] [Ideas]
**Is your feature request related to a problem? Please describe.**
There are a number of problems with the current feature system as far as I can see.

1. Products with a great amount of features become very, very unclear. It is very hard to distinguish the features form one another as the list does not have any way of grouping features together. So you end up with this situation where you can not separate features in any way. Therefore I suggest grouping features together in 'groups', this will make it much easier to manage.

2. Make some type of quick edit. Now an admin has to open every single feature value and edit it for the respective language. The worst part is that the language you are using to edit it isn't even persistent, thus making it very hard to create a lot of features and feature values as you have to scroll through the large list of languages every time you want to change / update something.

**Describe the solution you'd like**
1. It would be great if besides attributes and features, there would be another page for grouping features together. If the feature does not have a group, it will still show in the 'general' group. 

2. Make the feature value editable in the 'AdminFeatures' controller and add a language select box in the top right of the page (or somewhere else).

**Describe alternatives you've considered**
There are no alternatives to this problem. The only thing I found were very unwieldy addons on the marketplace.

**Additional context**
We are currently migrating to PrestaShop but this is one of the hurdles we stumbled upon. We are already making core changes (yes, because overriding a few hundred files is even worse) just to mitigate these problems and we are pretty sure that we are not the only ones that are affected by these issues.

**Comment by khouloudbelguith** [comment]
Hi @harm-smits,

Thanks for your report.
>Now an admin has to open every single feature value and edit it for the respective language. The worst part is that the language you are using to edit it isn't even persistent, thus making it very hard to create a lot of features and feature values as you have to scroll through the large list of languages every time you want to change / update something.

About this issue, we are aware of this issue, it is added to our bug roadmap: https://github.com/PrestaShop/PrestaShop/issues/13751

Ping @marionf @colinegin what do you think of his suggestion? is that possible to add another Tab page for grouping features together?

Thanks!
